<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>648</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>584</width>
    <height>618</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5840</width>
    <height>6180</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>583</width>
    <height>625</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>文件遍历器     powered by 高秀敏</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>561</width>
      <height>381</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>561</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>搜索文件</string>
    </property>
    <widget class="QCheckBox" name="displaySuffixBtn">
     <property name="geometry">
      <rect>
       <x>196</x>
       <y>65</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>是否展示后缀名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>68</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>选   项：</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="isRecursionFolder">
     <property name="geometry">
      <rect>
       <x>58</x>
       <y>65</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>是否递归遍历目录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchBtn">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>121</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>点我搜索文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="chooseBtn">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>请选择文件夹</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dirPath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>561</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>文本处理</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>32</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>关键字：</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="isScreenOutKeyword">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>32</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>是否筛除关键字</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>66</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>删除字符：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dealContent">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>60</y>
       <width>111</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>点我处理文本</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="keyword">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>27</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入关键字进行筛选或筛除(默认为筛选)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="needDelContent">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>62</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入需要删除的字符，多个字符串用多行文本表示</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>584</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
